pcl_filter_3d_utilities/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>pcl_filter_3d_utilities</name>
+  <version>0.1.0</version>
+  <description>Implements Various Pointcloud Filtering Services</description>
+  <maintainer email="example@example.com">Umass HRI Lab/Nerve Center Staff</maintainer>
```

Comment:
Maybe delete the email

Also change the maintainer to 
"UMass Lowell HRI Lab/NERVE Center"